website/source/docs/enterprise/vcs/bitbucket.html.md

```diff
@@ -0,0 +1,96 @@
+
+Upon saving the application, you will be redirected to https://bitbucket.org/account/user/your-username/api. Scroll down to OAuth Consumers and click on the application you just created. Copy the Key and Secret. Leave this tab open in your browser as you will need to return to it in a moment.
+
+### Creating a Terraform Enterprise OAuth Client
+
+In a new tab, navigate to https://atlas.hashicorptest.com/settings and, in the left-side panel, select the organization that you’d like to administer your Bitbucket connection, then click on “configuration” in the left-side panel.
+
+Within the “Add OAuthClient” pane, select Bitbucket Cloud and fill in your application key and secret. In the base url field, enter the root url of your Bitbucket instance (i.e. https://bitbucket.org). In the API url field, enter the base api url (i.e. https://api.bitbucket.org/2.0). Create the OAuth client.
+
+Once you have created your client, you will be redirected back to the configurations page for your chosen organization. On that page, find the “OAuth Clients” pane and copy the Callback URL for your Bitbucket Cloud OAuth Client. In the open Bitbucket tab, select the Terraform Enterprise OAuth Application and click edit. Enter the Callback URL you just copied in the field labeled Redirect URI. Save the application.
+
+Your OAuth Client should now be enabled for your Organization to use within Terraform Enterprise.
+
+## Using Terraform Enterprise with BitBucket Cloud
+
+There are two ways to connect your preferred VCS Host to Terraform Enterprise.
+You can generate an OAuth token both at the user and organization level.
+
+### Linking your Terraform Enterprise Organization
+
+Return to the settings page for the organization in which you created the OAuth Client (https://atlas.hashicorp.com/settings/organizations/your-organization/configuration). Find the section entitled Organization Connections to OAuth Client and click connect beneath your Bitbucket Cloud integration. You will be briefly redirected to Bitbucket in order to authenticate the client.
+
+Once you are redirected back to Terraform Enterprise, you should see that the token was created with a unique identifier. If you don’t, check the values in your OAuth Client and make sure they match exactly with the values associated with your Bitbucket OAuth Application. There is also an option to destroy the token and disconnect the organization from your Bitbucket installation.
+
+You are now ready to use your organization's token to manage builds and configurations within Terraform Enterprise.
+
+### Linking your Terraform Enterprise User Account
+
+Navigate to https://atlas.hashicorptest.com/settings/connections and click on “Connect Bitbucket Cloud to Atlas”. You will briefly be redirected to Bitbucket in order to authenticate your OAuth Client. Once redirected back to Terraform Enterprise, You should see a green flash banner with the message: "Successfully Linked to Bitbucket".
```

Comment:
Not all the links are clickable, so not sure if it would be clearer or more confusing. i believe if we don't have a title in the link, but it's regular link syntax -- they'll still be clickable!